package/MDAnalysis/analysis/base.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, trajectory, verbose=False, **kwargs):
         self._trajectory = trajectory
         self._verbose = verbose
 
-    def _setup_frames(self, trajectory, start=None, stop=None, step=None):
+    def _setup_frames(self, trajectory, start=None, stop=None, step=None, verbose = None):
         """
         Pass a Reader object and define the desired iteration pattern
         through the trajectory
```

Comment:
The docstring needs updating to reflect that `verbose` is now an explicit argument to `_setup_frames`. Please also include a `.. versionchanged` entry detailing that the `verbose` argument was added in 1.0.0.